### PR TITLE
docs: Remove redundant example code

### PR DESCRIPTION
**Pull request by pascal-hofmann**
The code also generates curve25519 v4 keys at the moment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -282,7 +282,7 @@ rsaKeyHigh, err := keyGenHandle.GenerateKeyWithSecurity(constants.HighSecurity)
 keyGenHandle = pgpDefault.KeyGeneration().AddUserId(name, email).New()
 // Generates curve25519 v4 keys.
 ecKey, err := keyGenHandle.GenerateKey()
-// Generates curve448 v4 keys.
+// Also generates curve25519 v4 keys.
 ecKeyHigh, err := keyGenHandle.GenerateKeyWithSecurity(constants.HighSecurity)
```
Comment (twiss) [inline README.md:286]:
```suggestion
```

**Review by twiss — COMMENTED** [review]
Hey :wave: Good catch! However, probably it would make more sense to just remove this example for now (since it's pretty useless as-is), until we can make it true and generate v4 Curve448 keys. RFC9580 (which introduces Curve448) is out now, but we might still want to wait a bit for support to become more widespread.

File: README.md (hunk 1 above)
Comment (pascal-hofmann) [reply README.md]: I removed the example.

**Review by twiss — APPROVED** [review]
:+1: Thanks!

**Pull request merged**